x-pack/legacy/plugins/siem/public/components/page/network/network_top_countries_table/translations.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const UNIT = (totalCount: number) =>
+  i18n.translate('xpack.siem.networkTopCountriesTable.unit', {
+    values: { totalCount },
+    defaultMessage: `{totalCount, plural, =1 {Country} other {Countries}}`,
+  });
+
+export const COUNTRY = i18n.translate('xpack.siem.networkTopCountriesTable.column.sourceIpTitle', {
+  defaultMessage: 'COUNTRY',
+});
+
+export const BYTES_IN = i18n.translate('xpack.siem.networkTopCountriesTable.column.bytesInTitle', {
+  defaultMessage: 'Bytes in',
+});
+
+export const BYTES_OUT = i18n.translate(
+  'xpack.siem.networkTopCountriesTable.column.bytesOutTitle',
+  {
+    defaultMessage: 'Bytes out',
+  }
+);
+
+export const FLOWS = i18n.translate('xpack.siem.networkTopCountriesTable.flows', {
+  defaultMessage: 'Flows',
+});
+
+export const DESTINATION_COUNTRIES = i18n.translate(
+  'xpack.siem.networkTopCountriesTable.destinationCountries',
+  {
+    defaultMessage: 'Top Destination Countries',
+  }
+);
+
+export const SOURCE_COUNTRIES = i18n.translate(
+  'xpack.siem.networkTopCountriesTable.sourceCountries',
+  {
+    defaultMessage: 'Top Source Countries',
+  }
+);
+
+export const DESTINATION_IPS = i18n.translate(
+  'xpack.siem.networkTopCountriesTable.destinationIps',
+  {
+    defaultMessage: 'Destination IPs',
+  }
+);
+
+export const SOURCE_IPS = i18n.translate('xpack.siem.networkTopCountriesTable.sourceIps', {
+  defaultMessage: 'Source IPs',
+});
+
+export const ROWS_5 = i18n.translate('xpack.siem.networkTopNFlowTable.rows', {
+  values: { numRows: 5 },
+  defaultMessage: '{numRows} {numRows, plural, =0 {rows} =1 {row} other {rows}}',
+});
+
+export const ROWS_10 = i18n.translate('xpack.siem.networkTopNFlowTable.rows', {
```

Comment:
These two keys will need updated or they'll collide when generating the i18n files.